test/rails5.0/app/cells/formtastic_cell.rb

```diff
@@ -1,4 +1,6 @@
 class FormtasticCell < Cell::ViewModel
+  view_paths << "#{Rails.root}/app/cells"
```

Comment:
great... maybe the rails app isn't requiring something, I'm gonna check, thanks for the feedback! 👍